sidecar/internal/csiaddonsnode/csiaddonsnode.go

```diff
@@ -85,15 +93,9 @@ type Manager struct {
 }
 
 // Deploy creates CSIAddonsNode custom resource with all required information.
-// When information to create the CSIAddonsNode is missing, an error will be
-// returned immediately. If creating the CSIAddonsNode in the Kubernetes
-// cluster fails (missing CRD, RBAC limitations, ...), an error will be logged,
-// and creation will be retried.
-func (mgr *Manager) Deploy() error {
-	object, err := mgr.getCSIAddonsNode()
-	if err != nil {
-		return fmt.Errorf("failed to get csiaddonsNode object: %w", err)
-	}
+// If creating the CSIAddonsNode in the Kubernetes cluster fails (missing CRD, RBAC limitations, ...)
+// an error will be logged and creation will be retried.
+func (mgr *Manager) Deploy(object *csiaddonsv1alpha1.CSIAddonsNode) error {
```

Comment:
can we also change `Deploy` to `deploy` as its not used outside of this package?